pkg/image/api/validation/validation.go

```diff
@@ -56,7 +56,7 @@ func validateImage(image *api.Image, fldPath *field.Path) field.ErrorList {
 	if len(image.DockerImageReference) == 0 {
 		result = append(result, field.Required(fldPath.Child("dockerImageReference"), ""))
 	} else {
-		if _, err := api.ParseDockerImageReference(image.DockerImageReference); err != nil {
+		if _, err := api.ParseMultiSegmentsDockerImageReference(image.DockerImageReference); err != nil {
```

Comment:
@miminar I'll do more tests when this approach will be approved by @liggitt, and @smarterclayton and you, because I'm not sure that we want to make these changes.